src/settings/Settings.js

```diff
@@ -120,7 +120,7 @@ export const SETTINGS = {
     },
     "feature_reactions": {
         isFeature: true,
-        displayName: _td("React to messages with emoji"),
+        displayName: _td("React to messages with emoji (refresh to apply changes)"),
```

Comment:
I understand the reason why we need to require a refresh, but also feel compelled to express my dissatisfaction with apps that require users to do this. I honestly think it is worth the engineering time to have changes apply live to provide a better user experience and inspire confidence from users. 

How feasible is it to have MatrixClient support a boolean flag with `setUnstableRelationAggregation` or something to change it? If it's far too difficult, then I'll just quietly protest instead of petition for changes.

Reply:
avoiding the refresh should not be high on our todo list, imo, relative to making the feature work

Reply:
I agree it's not great, but in this case it would be a bit annoying to make it work. We'd need to loop back over all the events and re-process any relations. Again, not super hard, but also unclear whether it's worth the trouble just to support flipping the feature on without refresh.